browse/controllers/abs_page/__init__.py

```diff
@@ -73,7 +74,8 @@ def get_abs_page(arxiv_id: str,
         abs_meta = metadata.get_abs(arxiv_id)
         response_data['abs_meta'] = abs_meta
         response_data['meta_tags'] = meta_tag_metadata(abs_meta)
-        response_data['author_links'] = queries_for_authors(abs_meta.authors)
+        response_data['author_links'] = \
+            split_long_author_list(queries_for_authors(abs_meta.authors), truncate_author_list_size)
```

Comment:
Unless I had this as an example, I would have trouble figuring out how to invoke this function. Can the function signature be simplified?